pyproject.toml

```diff
@@ -39,3 +39,40 @@ skip_glob = [
     "mypyc/test-data/*",
     "test-data/*",
 ]
+
+[tool.ruff]
+line-length = 99
+target-version = "py37"
+
+extend-ignore = [
+  "B006",  # use of mutable defaults in function signatures
```

Comment:
I think bugbear rules are off by default, can you explicitly use `select` to turn them on?

Also why `extend-ignore`? We're not extending any previous configuration, so should just be `ignore`

Reply:
Good point. I copied snippets of the config from another project which used `extend-ignore`, so that would be why I used `extend-ignore` instead of `ignore`.

Ruff does have a [default configuration](https://beta.ruff.rs/docs/configuration/#using-pyprojecttoml), though it seemingly doesn't select the `E` and `F` error codes like the docs say, since I had to enable them myself. Either way being explicit is better.